### ISSUE TITLE
Fix false positives for system-ui font-family-name-quotes

### ISSUE THREAD

**Issue by ScottSmith95** [stylelint/stylelint#3454]
> Clearly describe the bug

`system-ui` is a [standard](https://caniuse.com/#feat=font-family-system-ui) font keyword that should be exempted from the `font-family-name-quotes` rule.

> Which rule, if any, is the bug related to?

`font-family-name-quotes`

> What CSS is needed to reproduce the bug?

```css
body {
font-family: system-ui, -apple-system, 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif; }
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
	"rules": {
		"font-family-name-quotes": "always-unless-keyword",
	}
}
```

> Which version of stylelint are you using?

`9.3.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`$ stylelint 'styles/build/*.css'`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No. See: https://drafts.csswg.org/css-fonts-4/#system-ui-def

> What did you expect to happen?

`system-ui` should be treated as other system font keywords as referenced in #1295.

> What actually happened (e.g. what warnings or errors did you get)?

e.g. "The following warnings were flagged:"

```shell
Expected quotes around "system-ui"   font-family-name-quotes
```

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

I think it's just a matter of adding it to the [list of known keywords](https://github.com/stylelint/stylelint/blob/ffaa92a2503e0c99e9a2f9aecdff287c0d271b26/lib/reference/keywordSets.js#L81-L87). Consider contributing this change, please.

**Comment by ScottSmith95** [comment]
I'd be fine to do that or is the proper place to add this in the `isSystemFontKeyword()` function here? https://github.com/stylelint/stylelint/blob/master/lib/rules/font-family-name-quotes/index.js#L18 
Should tests be updated as well or are they fine as-is?

**Comment by hudochenkov** [comment]
`isSystemFontKeyword` is a workaround for non-standard keywords. Standard keywords are going to list I pointed above. And it would be picked up in other rules, which use this list.

This new keyword would fit nicely into [this test](https://github.com/stylelint/stylelint/blob/master/lib/rules/font-family-name-quotes/__tests__/index.js#L78).
